packages/components/src/divider/types.ts

```diff
@@ -24,6 +24,20 @@ export interface OwnProps {
 	marginEnd?: SpaceInput;
 }
 
+interface SeparatorPropsWithAdditionalDescription {
```

Comment:
Too much? Should I just put it inside `OwnProps`?

Reply:
Yeah, I'd personally add it to `OwnProps`. For added clarity, we could at most add a comment where we `omit` the prop from `SeparatorProps` to clarify that we're redefining for better docs? 

Alternatively, we could override it just in Storybook via `argTypes` (although I think I'd prefer the TypeScript solution)

Reply:
Agreed. I guess it's pretty self-evident why it's being overridden.